components/postgresql/postgresql.app.mjs

```diff
@@ -1,11 +1,215 @@
+import pg from "pg";
```

Comment:
I would also suggest using [pg-format lib](https://www.npmjs.com/package/pg-format) to also prevent potential SQL Injection. It is mentioned [here as well](https://node-postgres.com/features/queries#parameterized-query)